.env.docker.example

```diff
@@ -5,6 +5,8 @@ FALLBACK_COOKIE_DOMAIN=localhost
 
 DEBUG=true
 
+PHP_IDE_CONFIG=serverName=saproto-debug
```

Comment:
Does this take in to account the docker name variable?

Reply:
No, this is the name you give the debug server in your IDE, you just have to make sure that the two match

Reply:
Maybe we should add that to the README?

Reply:
wow, so smart